Make segment groups in segment groups really optional

Even though segment groups in segment groups are optional we still needs [this line](https://github.com/Hochfrequenz/ahbicht/blob/b2be34e62b3971a4740871b13672efdc70eccd13/unittests/test_validation.py#L90) otherwise you'll get an error:
```
    SegmentGroup(
E   TypeError: __init__() missing 1 required keyword-only argument: 'segment_groups'
```

--- a/src/maus/models/edifact_components.py
+++ b/src/maus/models/edifact_components.py
@@ -290,9 +290,12 @@ class SegmentGroup(SegmentLevel):
                 member_validator=attrs.validators.instance_of(Segment),
                 iterable_validator=attrs.validators.instance_of(list),
             )
-        )
+        ),
+        default=None,
     )  #: the segments inside this very group
-    segment_groups: Optional[List["SegmentGroup"]]  #: groups that are nested into this group
+    segment_groups: Optional[List["SegmentGroup"]] = attrs.field(
+        default=None
+    )  #: groups that are nested into this group
 
 
 class SegmentGroupSchema(SegmentLevelSchema):

--- a/unittests/test_edifact_components.py
+++ b/unittests/test_edifact_components.py
@@ -262,3 +262,15 @@ class TestEdifactComponents:
     def test_edifact_stack_to_from_json_path(self, json_path: str):
         stack = EdifactStack.from_json_path(json_path)
         assert stack.to_json_path() == json_path
+
+    def test_segment_group_can_be_instantiated_without_explicitly_defining_sub_groups(self):
+        """
+        Tests https://github.com/Hochfrequenz/mig_ahb_utility_stack/issues/41
+        """
+        sg = SegmentGroup(
+            discriminator="Foo",
+            ahb_expression="",
+        )  # must not throw an exception
+        assert sg is not None
+        assert sg.segment_groups is None
+        assert sg.segments is None

